samples/azure-quantum/hidden-shift-gallery/hidden-shift.ipynb

```diff
@@ -0,0 +1,1085 @@
+              "meas: 4/══════════════════════════════════════╩══╩══╩══╩═\n",
+              "                                              0  1  2  3 </pre>"
+            ],
+            "text/plain": [
+              "        ┌─────────┐   ┌──────────┐   ┌───┐ ░ ┌─┐         \n",
+              "   q_0: ┤ Ry(π/2) ├─■─┤ Ry(-π/2) ├─■─┤ H ├─░─┤M├─────────\n",
+              "        └──┬───┬──┘ │ └──┬───┬───┘ │ ├───┤ ░ └╥┘┌─┐      \n",
+              "   q_1: ───┤ H ├────■────┤ H ├─────■─┤ H ├─░──╫─┤M├──────\n",
+              "        ┌──┴───┴──┐   ┌──┴───┴───┐   ├───┤ ░  ║ └╥┘┌─┐   \n",
+              "   q_2: ┤ Ry(π/2) ├─■─┤ Ry(-π/2) ├─■─┤ H ├─░──╫──╫─┤M├───\n",
+              "        └──┬───┬──┘ │ └──┬───┬───┘ │ ├───┤ ░  ║  ║ └╥┘┌─┐\n",
+              "   q_3: ───┤ H ├────■────┤ H ├─────■─┤ H ├─░──╫──╫──╫─┤M├\n",
+              "           └───┘         └───┘       └───┘ ░  ║  ║  ║ └╥┘\n",
+              "meas: 4/══════════════════════════════════════╩══╩══╩══╩═\n",
+              "                                              0  1  2  3 "
+            ]
+          },
+          "execution_count": 8,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "# Create an object that represents IonQ's simulator target, \"ionq.simulator\".\n",
+        "# Note that any target you have enabled in this workspace can\n",
+        "# be used here. Azure Quantum makes it extremely easy to submit\n",
+        "# the same quantum program to different providers. \n",
+        "ionq_backend = provider.get_backend(\"ionq.simulator\")\n",
+        "\n",
+        "transpiled = transpile(circ, ionq_backend)\n",
+        "transpiled.draw()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we are submitting the transpiled circuit to the selected backend for 100 shots.\n",
+        "We'll get back a `job` instance and print it's job ID."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 9,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Job id: b207c958-9df2-11ec-802e-acde48001122\n"
+          ]
+        }
+      ],
+      "source": [
+        "# Using the IonQ simulator target, call \"run\" to submit the job. We'll\n",
+        "# use 100 shots (simulated runs).\n",
+        "job = ionq_backend.run(transpiled, shots=100)\n",
+        "print(\"Job id:\", job.id())"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The job ID can be used to retrieve the results later using the [get_details function](https://docs.microsoft.com/en-us/azure/quantum/optimization-job-reference#jobdetails) or by viewing it under the **Job management** section of the portal.\n",
```

Comment:
I took this text snippet from the hello world samples.

Reply:
Makes sense, and the target of the link appears to be correct, but it looks like there may be a bug in our docs publishing pipeline that's causing some Python docs to appear under optimization instead of QC. Not related to your PR, to be sure, so will follow up with the azure-quantum SDK folks.